datex2/pom.xml

```diff
@@ -96,6 +97,7 @@
 				<configuration>
 					<excludedGroups>broken</excludedGroups>
 				</configuration>
+                <version>${maven.surefire.plugin.version}</version>
```

Comment:
The indentation looks wrong here.

Reply:
addressed in 1fe5940d667da0881ad8595be524a396dc69a0e0